test/library/standard/Heap/assignment/heapAssignment.chpl

```diff
@@ -0,0 +1,8 @@
+use Heap;
+use Sort;
+var h1 = new heap(int);
+h1.push(2);
+h1.push(3);
+h1.push(1);
+var h2:heap(int) = h1;
+writeln(h2);
```

Comment:
Is this actually testing assignment?  I think line 3 is just initialization, and line 7 is copy-initialization.  I think to test assignment like the filename suggests you'd need to assign to a heap that's already initialized.